src/server/json_family.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021, Roman Gershman.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include "server/common.h"
+#include "server/engine_shard_set.h"
+
+namespace dfly {
+
+class ConnectionContext;
+class CommandRegistry;
+using facade::OpResult;
+using facade::OpStatus;
+
+class JsonFamily {
+ public:
+  static void Register(CommandRegistry* registry);
+
+ private:
+  static void Get(CmdArgList args, ConnectionContext* cntx);
+
+  static OpResult<std::string> OpGet(const OpArgs& op_args, std::string_view key, std::string_view path);
```

Comment:
Please move this function to the anonymous namespace in the cc file.

Reply:
Removed the OpGet function from the header file, and added this function into the anonymous namespace in the cc file.